### ISSUE TITLE
Yahoo Finance - Buy/Sell notification based on moving averages

### ISSUE THREAD

**Issue by jravenel** [jupyter-naas/awesome-notebooks#261]
# Yahoo Finance - Buy/Sell notification based on moving averages
Objective: Get a daily notification on a list of tickers to suggest buy or sell

## Input 
- tickers : ["AAPL", "FB", "TSLA", "MSFT"]
- start date: -360
- end date: today

## Model 
- get data with yahoo finance driver
- calculate 20MA and 50MA 
- if 20MA cross 50MA then generate buy notification 
- if 50MA cross 20MA then generate sell notification 

## Output
- schedule notebook daily to take 
- use naas notification to send daily email on allocation suggested in $ value

**Comment by jravenel** [comment]
Hello @vatsalkshah, as per our email discussion here is the issue. Keep me posted if you need support.

**Comment by FlorentLvr** [comment]
Hello @Gaurav1401 :) !
As discussed together, you can start working on this issue. Let me know if you have any issue.
Try to commit as much as possible that I will follow your code journey ! 🙏

**Comment by Dr0p42** [comment]
> @FlorentLvr Sir, Do I need to share any visualizations or graph in the mail?

Hello @Gaurav1401, I think that it would be interesting to provide some insight and help the person that will get the notification to indeed buy or sell.

What do you think?

**Comment by Gaurav1401** [comment]
> > @FlorentLvr Sir, Do I need to share any visualizations or graph in the mail?
> 
> Hello @Gaurav1401, I think that it would be interesting to provide some insight and help the person that will get the notification to indeed buy or sell.
> 
> What do you think?

I agree with you @Dr0p42 Sir, but I don't understand the logic, if mean of 20MA > 50MA then generate a buy notification and vice-versa.
and using how to represent this 20MA and 50MA in a plot.

**Comment by Dr0p42** [comment]
> > > @FlorentLvr Sir, Do I need to share any visualizations or graph in the mail?
> > 
> > 
> > Hello @Gaurav1401, I think that it would be interesting to provide some insight and help the person that will get the notification to indeed buy or sell.
> > What do you think?
> 
> I agree with you @Dr0p42 Sir, but I don't understand the logic, if mean of 20MA > 50MA then generate a buy notification and vice-versa. and using how to represent this 20MA and 50MA in a plot.

@FlorentLvr I let you answer this one :)

**Comment by FlorentLvr** [comment]
> > > > @FlorentLvr Sir, Do I need to share any visualizations or graph in the mail?
> > > 
> > > 
> > > Hello @Gaurav1401, I think that it would be interesting to provide some insight and help the person that will get the notification to indeed buy or sell.
> > > What do you think?
> > 
> > 
> > I agree with you @Dr0p42 Sir, but I don't understand the logic, if mean of 20MA > 50MA then generate a buy notification and vice-versa. and using how to represent this 20MA and 50MA in a plot.
> 
> @FlorentLvr I let you answer this one :)

Hey @Gaurav1401 ! I think you can just add the chart to show that 20MA crossed the 50MA. So you could do a email like:
"Hi there,
It's time to <ACTION> <STOCK> stock !
Please see the moving averages trend below:
<CHART_IMAGE>"

**Comment by Dr0p42** [comment]
Hello @Gaurav1401, any update on this issue? Let us know if we can help you out :)

**Comment by jravenel** [comment]
Moving it back to backlog @Gaurav1401 let us know when you are back.
